src/cljs/replumb/repl.cljs

```diff
@@ -744,6 +764,7 @@
   supporting:
 
   * :verbose - will enable the the evaluation logging, defaults to false
+  * :filename - the path to a file to load, if any
```

Comment:
Could we refactor out the things we need so that we do not modify `read-eval-call` here?